### ISSUE TITLE
Use "semver compatible" constraints for dependencies

### ISSUE THREAD

**Issue by triccardi-systran** [vsivsi/gridfs-locking-stream#3]
Current package.json:
```json
{
  "dependencies": {
    "gridfs-stream": ">= 0.5.3",
    "gridfs-locks": ">= 1.2.2"
  }
}
```
is invalid because current gridfs-locks-stream is probably not compatible with breaking changes that will occure in gridfs-locks 2.x, and gridfs-stream 1.x or 2.x...

It should probably be something like that, using "semver compatible" constraints:
```json
{
  "dependencies": {
    "gridfs-stream": "^0.5.3",
    "gridfs-locks": "^1.2.2"
  }
}
```

**Comment by vsivsi** [comment]
Good catch, I'm about to update gridfs-locking-stream, and I'll be sure to make this change.

**Comment by vsivsi** [comment]
Fixed in version v0.2.6.  FYI, a new major revision (v1.0) of this package will be released shortly with support for gridfs-stream 1.0.x and mongodb native driver 2.0.x.
